packages/eslint-config-react-app/index.js

```diff
@@ -26,13 +26,21 @@ module.exports = {
   plugins: ['import', 'flowtype', 'jsx-a11y', 'react'],
 
   env: {
-    browser: true,
     commonjs: true,
     es6: true,
     jest: true,
     node: true,
   },
 
+  globals: {
+    document: true,
```

Comment:
@gaearon `document` and `navigator` are not part of node, so it still can bring headache to SSR app developer.